src/proof/resolution_bitvector_proof.h

```diff
@@ -0,0 +1,102 @@
+/*********************                                                        */
+/*! \file resolution_bitvector_proof.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Liana Hadarean, Mathias Preiner, Guy Katz
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2018 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Bitvector proof
+ **
+ ** Bitvector proof
+ **/
+
+#include "cvc4_private.h"
+
+#ifndef __CVC4__RESOLUTION__BITVECTOR__PROOF_H
+#define __CVC4__RESOLUTION__BITVECTOR__PROOF_H
```

Comment:
Please use the format described in the [style guide](https://github.com/CVC4/CVC4/wiki/Code-Style-Guidelines#the-define-guard).

Reply:
The name still doesn't correspond to the style guide (needs underscores between the words).

Reply:
I believe this is outdated. Also, I don't think that the style guide says that there should be spaces between the words. Am I mis-reading it?